src/Carousel.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from "react";
+
+
+const images : string[] = [
+    "https://images.pexels.com/photos/45923/pexels-photo-45923.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+    "https://images.pexels.com/photos/10901851/pexels-photo-10901851.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+    "https://images.pexels.com/photos/19383822/pexels-photo-19383822/free-photo-of-red-double-decker-on-oxford-street-in-london-at-night.jpeg?auto=compress&cs=tinysrgb&w=800",
+    "https://images.pexels.com/photos/7245319/pexels-photo-7245319.jpeg?auto=compress&cs=tinysrgb&w=800",
+    "https://images.pexels.com/photos/19736357/pexels-photo-19736357/free-photo-of-london-bus-on-road-towards-big-ben.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=2",
+];
+
+const Carousel = () => {
+    const [currentIndex, setCurrentIndex] = useState(0);
+  
+    const handleNext = () => {
+        setCurrentIndex((prevIndex) =>
+        prevIndex + 1 === images.length ? 0 : prevIndex + 1
+      );
+    };
+    const handlePrevious = () => {
+      setCurrentIndex((prevIndex) =>
+        prevIndex - 1 < 0 ? images.length - 1 : prevIndex - 1
+      );
+    };
+    const handleDotClick = (index : number) => {
+      setCurrentIndex(index);
+    };
+    return (
+    <div className="carousel-images">
+        <img
+          key={currentIndex}
+          src={images[currentIndex]}
+        /><div className="slide_direction">
```

Comment:
Eslint has flagged an issue where the images are lacking alt text. Please could you review this ESlint error and resolve as you see fit.